src/main/java/org/usfirst/frc4904/robot/commands/Shoot.java

```diff
@@ -0,0 +1,34 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.subsystems.Indexer;
+import org.usfirst.frc4904.robot.subsystems.Shooter;
+import org.usfirst.frc4904.robot.subsystems.Flywheel.FlywheelStatus;
+import org.usfirst.frc4904.standard.commands.RunIf;
+import org.usfirst.frc4904.standard.commands.WaitUntil;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+public class Shoot extends SequentialCommandGroup {
+  protected static final double SPEED_TOLERANCE = 0.001; // TODO: Untested value
+  /**
+   * Check that the flywheel is spun up, spin up if it isn't, then call IndexOne
+   * 
+   * Psudo code:  
```

Comment:
remove pseudo code?

Reply:
Was intended to tell a user what this does in detail because the logic is somewhat complicated. I can remove it if you want.

Reply:
ok thats fine: psudo is spelled pseudo